packages/react-toast/src/components/Toast/index.tsx

```diff
@@ -23,39 +24,38 @@ export const Toast: React.FunctionComponent<{
     throw new Error("Missing Courier Provider");
   }
 
-  const { clientKey, transport } = courierContext;
-  const courierActions = useActions();
+  const { clientKey, transport, dispatch } = courierContext;
 
   const handleToast = useCallback(
     (message: ICourierToastMessage | string) => {
       message =
         typeof message === "string"
           ? ({
-            body: message,
-            icon: undefined,
-          } as ICourierToastMessage)
+              body: message,
```

Comment:
extra spacing

Reply:
we need to get the linter right on this stuff.